src/universal/mutations/StartNewMeetingMutation.js

```diff
@@ -0,0 +1,86 @@
+import {commitMutation} from 'react-relay';
+import {meetingTypeToSlug, phaseTypeToSlug} from 'universal/utils/meetings/lookups';
+
+graphql`
+  fragment StartNewMeetingMutation_team on StartNewMeetingPayload {
+    team {
+      id
+      meetingId
+      newMeeting {
+        id
+        createdAt
+        facilitatorStageId
+        facilitatorUserId
+        meetingNumber
+        meetingType
+        phases {
+          stages {
```

Comment:
See the comment in NewMeetingPhase.js. That allows us to get isComplete here without having to make a new inline fragment to grab it